api/preprints/serializers.py

```diff
@@ -150,17 +160,17 @@ def update(self, preprint, validated_data):
             preprint.node.preprint_article_doi = validated_data['article_doi']
             save_node = True
 
+        if 'license_type' in validated_data or 'license' in validated_data:
```

Comment:
The datacite metadata has a spot for license - since the call to create a new EZID happens in publish, I wanted to set the license if it's there before the call to publish so that datacite and EZID could store the license information in the metadata